lib/screens/v2/widget_instance/subway_status.ex

```diff
@@ -336,112 +415,166 @@ defmodule Screens.V2.WidgetInstance.SubwayStatus do
-        serialize_green_line_trunk_alert(trunk_alert, statuses, alert_count)
+        {[trunk_alert], []} ->
+          %{type: :extended, alert: serialize_trunk_alert(trunk_alert)}
 
-      _ ->
-        # If there are multiple alerts on the GL trunk, log it and serialize the count
-        _ = Logger.info("[subway_status_multiple_alerts] route=Green-Trunk count=#{alert_count}")
+        # If there is a single alert on the GL trunk, show it in its own row.
+        # Show branch alert/summary on another row.
+        {[trunk_alert], branch_alerts} ->
+          %{
+            type: :contracted,
+            alerts: [
+              serialize_trunk_alert(trunk_alert),
+              serialize_green_line_branch_alerts(branch_alerts, true)
+            ]
+          }
 
-        %{
-          type: :single,
-          route: serialize_route_pill("Green"),
-          status: "#{alert_count} alerts",
-          location: %{full: "mbta.com/alerts/subway", abbrev: "mbta.com/alerts/subway"}
-        }
+        # If the only 2 GL alerts are on the trunk, show them both.
+        {[trunk_alert1, trunk_alert2], []} ->
+          %{
+            type: :contracted,
+            alerts: [serialize_trunk_alert(trunk_alert1), serialize_trunk_alert(trunk_alert2)]
+          }
+
+        # 2+ trunk alerts w/ 1+ branch alert or 3+ trunk alerts
+        # show alert count summary
+        {_, _} ->
+          %{
+            type: :contracted,
+            alerts: [serialize_alert_summary(alert_count, serialize_route_pill("Green"))]
+          }
+      end
     end
   end
 
-  defp serialize_green_line_trunk_alert(trunk_alert, statuses, alert_count) do
-    case statuses do
-      [] ->
-        # One GL trunk alert and no branch alerts
-        Map.merge(
-          %{type: :single, route: serialize_route_pill("Green")},
-          serialize_alert(trunk_alert, "Green")
-        )
-
-      [status] ->
-        # One GL trunk alert and one GL branch alert
-        trunk_status = get_green_line_branch_status(trunk_alert)
-        %{type: :multiple, statuses: [[nil, trunk_status], status]}
-
-      _ ->
-        # One GL trunk alert and 2+ GL branch alerts
-        _ = Logger.info("[subway_status_multiple_alerts] route=Green-Trunk count=#{alert_count}")
+  defp serialize_trunk_alert(alert) do
+    Map.merge(
+      %{route_pill: serialize_route_pill("Green")},
+      serialize_alert(alert, "Green")
+    )
+  end
 
-        %{
-          type: :single,
-          route: serialize_route_pill("Green"),
-          status: "#{alert_count} alerts",
-          location: %{full: "mbta.com/alerts/subway", abbrev: "mbta.com/alerts/subway"}
-        }
-    end
+  defp serialize_alert_summary(alert_count, route_pill) do
+    %{
+      route_pill: route_pill,
+      status: "#{alert_count} current alerts",
+      location: %{full: "mbta.com/alerts/subway", abbrev: "mbta.com/alerts/subway"}
```

Comment:
mbta.com/alerts now redirects to /alerts/subway, and also the designs use /alerts, so I believe this should be just the plain string `"mbta.com/alerts"`, no full/abbrev map or longer url needed.